modules/ROOT/pages/platform/security/secure-connections.adoc

```diff
@@ -7,24 +7,52 @@
+AuraDB Virtual Dedicated Cloud and AuraDS Enterprise run in a dedicated AWS cloud account, Azure subscription or GCP project to achieve complete isolation for your deployment.
+Additional Virtual Private Cloud (VPC) boundaries enable you to operate within an isolated section of the service, where your processing, networking, and storage are further protected. 
+The Aura console resides in a separate VPC, isolated from the rest of the Aura services.
 
-Additional VPC boundaries enable you to operate within an isolated section of the service, where your processing, networking, and storage are further protected.
+== Network access
 
-The Aura Console runs in a separate VPC, separate from the rest of Aura.
+An Aura instance can be publicly available, completely private, or configured to allow both public and private access.
+To configure network access, you need to be authorized to access the part of [your system's?] infrastructure that runs and handles these instances as well as the networking used to establish secure connections between the database and the application's VPC. This includes the ability to connect over the cloud provider's private link and private endpoint.
 
-== Network access
+=== Configure network access
 
-An Aura instance can be publicly available, completely private, or both.
-To configure this, you need to be authorized to access the part of the infrastructure that runs and handles these instances as well as the networking used to establish secure connections between the database and the application's VPC.
-This includes the ability to connect over the cloud provider's private link and private endpoint.
+To configure settings for network access to your instance go to *Aura console* > *Security* > *Network access* > *New network access configuration*
+From there, you can either set up a new network access configuration, or edit current configuration settings.
 
-If your Aura instances are public, traffic to them is allowed to traverse the public internet and they are accessible with the correct username and password.
+The Aura console provides a step-by-step configuration guide to:
 
-For your instance to be completely private, turn public traffic off, use the cloud provider's network, and create a private endpoint inside your VPC, which gives you a private connection to Aura.
-The only way to connect to your database is from inside your network (your VPC in your AWS/Azure/GCP account) using an internal IP address you choose and DNS records you create.
+. Choose your Aura instance details
+. Create a VPC endpoint
+. Accept endpoint connection requests and enable private DNS in the cloud provider's console
+. Disable public traffic (optional). 
+If you disable public traffic it is highly recommended to link:https://support.neo4j.com/s/article/13174783967507-How-To-Test-Connectivity-Through-The-Private-Endpoint[Test Connectivity Through The Private Endpoint]
```

Comment:
I was wondering, if I should put the steps in here? They are available in a KB and it's a major pain point that customer's are not checking their connectivity.